core/templates/domain/exploration/OutcomeObjectFactory.ts

```diff
@@ -33,21 +33,22 @@ export interface OutcomeBackendDict {
   'feedback': SubtitledHtmlBackendDict;
   'labelled_as_correct': boolean;
   'param_changes': readonly ParamChangeBackendDict[];
-  'refresher_exploration_id': string;
-  'missing_prerequisite_skill_id': string;
+  'refresher_exploration_id': string | null;
+  'missing_prerequisite_skill_id': string | null;
```

Comment:
Why do we need to allow `null` here?

Reply:
And in the other places?

Reply:
In proof of this pr I have attached screenshot of state_domain.py file where it is written that refresher_exploration_id excepts both str or None.